<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tourism Agency</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>591</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_content"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>127</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_buttons">
     <item>
      <widget class="QPushButton" name="button_register_travel_pack">
       <property name="text">
        <string>Register travel pack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_register_plane_ticket">
       <property name="text">
        <string>Register plane ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_register_hosting">
       <property name="text">
        <string>Register hosting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_show_promo_packs">
       <property name="text">
        <string>Promo packages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_packages">
       <property name="text">
        <string>Packages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_plane_tickets">
       <property name="text">
        <string>Plane tickets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_hosting">
       <property name="text">
        <string>Hosting</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
